perf.go

```diff
@@ -40,8 +38,7 @@ func init() {
 	//blas64.Use(cgo.Implementation{})
```

Comment:
This needs to change. There is not `cgo.Implementation`.

Reply:
done.